src/app/Greeting.js

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from '../features/greeting/greetingSlice';
+
+function Greeting() {
+  const message = useSelector((state) => state.message.message);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchGreetings());
+  }, []);
+  fetchGreetings();
+  return (
+    <div className="App">
+      <h1>
+        {!message.loading && message.message}
+      </h1>
+    </div>
+  );
+}
+
+export default Greeting;
```

Comment:
- [ ] I cant see any message on the page. Kindly fix it.
![image](https://user-images.githubusercontent.com/38635158/185705493-c2b9212f-604f-4bd1-b071-e0ff91d888ac.png)
NOTE: I run the rails server and ı see the JSON file. The rails side is working.
![image](https://user-images.githubusercontent.com/38635158/185705664-e3bdb144-3256-4e23-8fd8-b25f64bdc423.png)